scanners/zap-automation-framework/examples/demo-zap-advanced-migrate-scan/scan.yaml

```diff
@@ -0,0 +1,93 @@
+# SPDX-FileCopyrightText: the secureCodeBox authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: "zap-automation-framework-migrate-advanced-scan-config"
```

Comment:
is this required to have migrate scan in the name?
wouldnt this be outdated quite quick after the migration has been completed?